maintainers/maintainer-list.nix

```diff
@@ -4671,6 +4671,11 @@
     github = "sigma";
     name = "Yann Hodique";
   };
+  sikmir = {
+    email = "sikmir@gmail.com";
+    github = "sikmir";
+    name = "Nikolay Korotkiy";
+  };
```

Comment:
IIRC, adding yourself as a maintainer is preferred to be a separate commit.

Something like, `maintainers: add sikmir`. Then apply the qtpbfimageplugin commit.

Other than that, `nix-review pr 63506` passed, and LGTM otherwise

Reply:
Thanks, fixed.